lms/static/js/verify_student/views/make_payment_step_view.js

```diff
@@ -30,6 +30,9 @@ var edx = edx || {};
                 currency: 'usd',
                 upgrade: false,
                 verificationDeadline: '',
+                // TODO: user prefs go here
+                userTimezone: 'UTC', 
+                userLanguage: 'en-US',
```

Comment:
I have no idea how we're going to get these values; documentation for "pass a value from a mako context variable through an underscore template it loads" is not readily available. We'll figure something out.